### PR TITLE
Add rice-ball-rolling's Scottsdate file

### LINKED ISSUE #22: rice-ball-rolling's workflow for adding Scottsdale to the Repository
## Scottsdale, AZ
- [x] Create an issue 
- [x] Create a branch 
- [x] Make changes
- [x] Open a PR
- [ ] Collaborate/Have discussion
- [ ] Get approval to :smile :smile:  
- [ ] Merge

### PR DESCRIPTION
**Pull request by rice-ball-rolling**
closes #22

### CODE REVIEW

**Review by rice-ball-rolling — COMMENTED** [review]
cool

File: githubteacher-rice-ball-rolling.md [hunk 1]
```diff
@@ -0,0 +1 @@
+TEST
```
Comment (rice-ball-rolling) [inline githubteacher-rice-ball-rolling.md]: xxx

Comment (rice-ball-rolling) [reply githubteacher-rice-ball-rolling.md]: xxx

**Review by rice-ball-rolling — COMMENTED** [review]
xxx

File: githubteacher-rice-ball-rolling.md (hunk 1 above)
Comment (rice-ball-rolling) [reply githubteacher-rice-ball-rolling.md]: I guess I am behind

**Review by odragu-processia — APPROVED** [review]

**Pull request merged**